### PR TITLE
fix grammatical issues in mobile accessibility

### PR DESCRIPTION
**Pull request by yashikabhargava**
### Description

fixed minor grammatical issues in mobile accessibility.

### CODE REVIEW

File: files/en-us/learn/accessibility/mobile/index.md [hunk 1]
```diff
@@ -262,14 +262,14 @@ Click here for a [good hamburger menu example](https://fritz-weisshart.de/meg_me
...
 
 - Types `number`, `tel`, and `email` display suitable virtual keyboards for entering numbers/telephone numbers.
 - Types `time` and `date` display suitable pickers for selecting times and dates.
 
-If you want to provide a different solution for desktops, you could always serve different markup to your mobile devices using feature detection. See [input types](https://diveinto.html5doctor.com/detect.html#input-types) for raw information on detecting different input types, and also check out our [feature detection article](/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) for much more information.
+If you want to provide a different solution for desktops, you could always serve different markups to your mobile devices using feature detection. See [input types](https://diveinto.html5doctor.com/detect.html#input-types) for raw information on detecting different input types, and also check out our [feature detection article](/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Feature_detection) for much more information.
```
Comment (hidde) [inline files/en-us/learn/accessibility/mobile/index.md:272]: I'd say markup is markup in plural form too, let's leave this as ‘markup’

**Review by hidde — CHANGES_REQUESTED** [review]

File: files/en-us/learn/accessibility/mobile/index.md (hunk 1 above)
Comment (yashikabhargava) [reply files/en-us/learn/accessibility/mobile/index.md:272]: Yes that is right, thank you for correcting my mistake.

**Comment by yashikabhargava** [comment]
@hidde I've made the change, kindly review my PR.

**Comment by hidde** [comment]
✨ thank you, looks good to me!

**Review by hidde — APPROVED** [review]

**Pull request merged**